docs/docs/0.4.0/documentation/http-api.md

```diff
@@ -1339,7 +1339,7 @@ curl http://localhost:5681/meshes/default/fault-injections/fi1
    "match": {
     "protocol": "http",
     "service": "frontend",
-    "version": "0.1"
+    "version": "0.1",
```

Comment:
I don't think this is a valid JSON, also check other places here

Reply:
Yep, thanks! A bit of Go deformation